Gemfile

```diff
@@ -19,6 +19,6 @@ gem 'rubocop', '~> 1.24'
 gem 'rubocop-rake'
 gem 'rubocop-rspec', '~> 2.1'
 gem 'rubyXL' # for updating Excel spreadsheets
+gem 'sdr-client', '~> 0.67.1'
```

Comment:
I would try `'~> 0.67` and then `bundle update` and see what version you get in `Gemfile.lock`.

Reply:
Since you get `0.67.1` with bundle update, I've changed this back to `'~> 0.67'`, thanks.